Makefile

```diff
@@ -179,6 +179,10 @@ lint:
 unit:
 	hack/unit.sh ${GO_MOD_FLAGS}
 
+.PHONY: community
+community:
```

Comment:
A more descriptive target name would be helpful
consider `community-bundle` or `community-manifests`